common/transport/amqp/src/amqp_cbs.ts

```diff
@@ -274,6 +285,7 @@ export class ClaimsBasedSecurityAgent {
             const links = [this._senderLink, this._receiverLink];
             async.each(links, (link, callback) => {
               if (link) {
+                debug('while detaching for  link ');
```

Comment:
>  link  [](start = 42, length = 7)

minor nit: you have extra spaces there.

Reply:
Not anymore.

---
In reply to: [209348926](https://github.com/Azure/azure-iot-sdk-node/pull/344#discussion_r209348926) [](ancestors = 209348926)